packages/components/board/stories/board.stories.js

```diff
@@ -5,14 +5,19 @@ import '@tradeshift/elements.board';
 
 storiesOf('ts-board', module)
 	.addDecorator(withKnobs)
-	.add('default', () => {
-		const dir = text('ltr', 'Dir');
-		const title = text('My Board', 'Title');
-		const content = text('Put any html inside the board', 'Content');
-
-		return html`
-			<ts-board dir="${dir}" data-title="${title}">
-				${content}
-			</ts-board>
-		`;
-	});
+	.add(
+		'default',
+		() => {
+			const dir = text('ltr', 'Dir');
```

Comment:
Can you please change this to a radio? We should refactor all elements and make them consistent.